test/create_visit_location_test.go

```diff
@@ -15,7 +15,7 @@ func TestCreatePlace(t *testing.T) {
 
 	expectedLatitude := 32.715736
 	expectedLongitude := -117.161087
-	place := POI.CreatePlace(name, microAddr, addr, "OPERATIONAL", "stay", nil, "landmark_mtv", 3, 4.5, "", nil, 0, expectedLatitude, expectedLongitude)
+	place := POI.CreatePlace(name, microAddr, addr, "OPERATIONAL", "stay", nil, "landmark_mtv", 3, 4.5, "", nil, 0, expectedLatitude, expectedLongitude, nil)
```

Comment:
it's better to add some test for a place with valid summary info

Reply:
Done.